<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>SEAStAR | Armbrust Lab</title>
    <meta name="author" content="  ">
    <meta name="description" content="Armbrust Lab, Biological Oceanography, University of Washington
">
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website">


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous">

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light">

    <!-- Styles -->
    
    <link rel="shortcut icon" href="/assets/img/favicon.ico">
    
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://armbrustlab.github.io/tools/seastar/">
    
    <!-- Dark Mode -->
    
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark">

    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
    

  </head>

  <!-- Body -->
  <body class="fixed-top-nav ">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="/">Armbrust Lab</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/"></a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item ">
                <a class="nav-link" href="/research/">research</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/people/">people</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/tools/">tools</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/contact/">contact</a>
              </li>

              <!-- Toogle theme mode -->
              <li class="toggle-container">
                <button id="light-toggle" title="Change theme">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </nav>

    </header>


    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">SEAStAR</h1>
            <p class="post-description"></p>
          </header>

          <article>
            <figure class="figure w-100 text-center">
  <a href="/assets/img/metagenome-large.gif">
    <img src="/assets/img/metagenome-small.gif" title="metagenome" class="figure-img img-fluid">
  </a>
</figure>

<p><strong>SEAStAR tools version 0.5.0 has been released on <a href="https://github.com/armbrustlab/SEAStAR" rel="external nofollow noopener" target="_blank">github</a></strong>.</p>

<p>SEAStAR is a package of open-source tools supporting the construction of analysis pipelines for environmental next-generation sequencing data. Version 0.4 includes new support for Illumina® reads in addition to SOLiD™ data, and provides high-performance tools for dealing with:</p>

<ul>
  <li>Converting between file formats (CSFASTA -&gt; FASTQ)</li>
  <li>Trimming raw reads for quality (with tuning support)</li>
  <li>PCR de-duplication of paired reads (without reference sequences)</li>
  <li>Selecting and estimating the relative abundance of sequences from large reference databases (e.g. 16S rDNA)</li>
  <li>Sub-sampling paired FASTQ files randomly, or based on reads included in (or excluded from) reference alignments</li>
  <li>Converting assembled color-space (SOLiD) contigs to nucleotide-space</li>
  <li>Connecting assembled contigs together via paired reads (constructing an assembly graph)</li>
  <li>Splitting complicated metagenomic assembly graphs into well-supported scaffolds</li>
  <li>Binning scaffolds by organism using tetra-nucleotide statistics</li>
  <li>Identifying small circular scaffolds that are likely virus or plasmid genomes</li>
</ul>

<p>The prototype version of SEAStAR was developed and used to analyze the metagenome data in our publication: “<a href="http://www.sciencemag.org/content/335/6068/587.abstract" rel="external nofollow noopener" target="_blank">Untangling Genomes from Metagenomes: Revealing an Uncultured Class of Marine Euryarchaeota</a>”.</p>

<p>It was also used extensively for the analysis in Vaughn Iverson’s 2015 Ph.D. dissertation “<a href="https://digital.lib.washington.edu/researchworks/handle/1773/35283" rel="external nofollow noopener" target="_blank">Untangling Genomes from Metagenomes</a>”.</p>

<p>SEAStAR was publicly released in the following phases (versions prior to 0.3 were prototypes):</p>

<h4 id="phase-1-ver-03">Phase 1 (ver 0.3):</h4>
<p>Phase 1 contained tools developed for working with SOLiD reads, converting them to colorspace FASTQ files, removing PCR duplicates, and trimming the reads based on quality and information content for various uses. These tools generate files that can be directly used with aligners and assemblers, such as <a href="http://bio-bwa.sourceforge.net/" rel="external nofollow noopener" target="_blank">BWA</a> and <a href="http://www.ebi.ac.uk/~zerbino/velvet/" rel="external nofollow noopener" target="_blank">Velvet</a>.</p>

<h4 id="phase-2-ver-04">Phase 2: (ver 0.4)</h4>
<p>Phase 2 added support for Illumina sequence, and contains the tools we developed for processing SAM alignment files and assembled nucleotide or colorspace contigs into mate-paired assembly graphs with associated nucleotide contigs that can then be visualized and explored using <a href="http://www.graphviz.org/" rel="external nofollow noopener" target="_blank">GraphViz</a>. This release also contains the tools for selecting and visualizing the best sequences from a reference database (e.g. the <a href="http://rdp.cme.msu.edu/" rel="external nofollow noopener" target="_blank">RDP 16S database</a>) alignment with associated statistics such as coverage and relative abundance. In the case of alignments with RDP, these sequences can then be classified (using the <a href="http://sourceforge.net/projects/rdp-classifier/" rel="external nofollow noopener" target="_blank">RDP classifier</a>) and visualized taxonomically. Also included are tools for splitting assembly graphs into parsimonious linear scaffold graphs, binning these scaffold graphs using tetra-nucleotide statistics, and constructing scaffold sequences with the order and orientation of contigs properly determined. Use of this full pipeline produces candidate genome assemblies ready for refinement, gap-filling (if desired) and annotation.</p>

<h4 id="phase-3-ver-05">Phase 3: (ver 0.5)</h4>
<p>Adds support for identifying, isolating and reassembling topologically circular (virus and plasmid) scaffold sequences.</p>

<p>Thanks for your interest in the SEAStAR tools, we hope you enjoy using them.
– <a href="/people/iverson/">Vaughn Iverson</a> and <a href="/people/berthiaume/">Chris Berthiaume</a></p>

<figure>

  <picture>
    
    <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/assets/img/SEAStAR_Pipeline_Stages-480.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/assets/img/SEAStAR_Pipeline_Stages-800.webp"></source>
    <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/assets/img/SEAStAR_Pipeline_Stages-1400.webp"></source>
    

    <!-- Fallback to the original file -->
    <img src="/assets/img/SEAStAR_Pipeline_Stages.png" class="img-fluid rounded z-depth-1" width="auto" height="auto" alt="SEAStAR pipeline stages" onerror="this.onerror=null; $('.responsive-img-srcset').remove();">

  </picture>

</figure>


          </article>

        </div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        © Copyright 2023 University of Washington. 
      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Masonry & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script>
  <script defer src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script defer src="/assets/js/common.js"></script>

    

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
    

  </body>
</html>
