---
title: Bryndan P. Durham
gallery:
  - url: /assets/images/cruise547.jpg
    image_path: /assets/images/cruise547.jpg
    alt: "Plankton collection"
    title: "Collection of plankton onboard the UW's R/V Thomas G. Thompson"
  - url: /assets/images/Metabolites.jpg
    image_path: /assets/images/Metabolites.jpg
    alt: "Metabolite extraction"
    title: "Extraction of plankton metabolites in the lab"
---
![Bryndan P. Durham]({% link /assets/images/DURHAM.jpg %})

## Research associate
email: bpdurham@uw.edu  
office: (206) 685-4196  


{% include figure image_path="/assets/images/diatoms.jpg" alt="Diatoms" caption="Diatoms under the microscope" %}{: .align-right}

My research focuses on how marine microbial interactions influence nutrient and energy cycles in the surface ocean.

Oceanic primary production, carried out predominantly by unicellular phytoplankton, generates one of the largest reservoirs of organic carbon on Earth. About half of this fixed carbon is degraded by heterotrophic bacteria within minutes to weeks, a transfer that accounts for the largest flux of carbon through the ocean. Yet, the diversity of organic molecules and interacting organisms present in seawater has hindered detection of specific relationships that mediate this large flux of energy and matter. 

To explore metabolite exchange in bacterial-phytoplankton interactions, I have employed model microbial system approaches where I co-culture marine bacteria and diatoms together and use gene expression and metabolite analyses to assay for compounds passed between them. Through these approaches, I have detected exchange of novel organosulfur molecules (sulfonates) between diatoms and bacteria. Genes for biosynthesis and degradation of these organic sulfur compounds have limited distribution among phytoplankton and bacterioplankton, respectively, suggesting that sulfonated substrates underlie targeted microbial interactions.

I am currently using a combination of laboratory- and field-based measurements to study sulfonates in terms of their contribution to marine organic matter flux, their taxonomically driven spatiotemporal dynamics, and their roles in ecosystem interdependencies.
{% include gallery caption="**1.** Collection of plankton onboard the UW's R/V Thomas G. Thompson. ; Photo credit: Robyn Von Swank. **2.** Extraction of plankton metabolites in the lab." %}

## Education
* Ph.D., Microbiology, University of Georgia, 2014
* B.S., Biology, Virginia Tech, 2008

## Publications
Moran MA, **Durham BP**. Sulfur metabolites in the pelagic ocean. *Nature Reviews Microbiology* accepted.

**Durham BP**, Boysen AK, Carlson LT, Groussman RD, Heal KR, Cain KR, Morales RL, Coesel SN, Morris RM, Ingalls AE, Armbrust EV. Sulfonate-based networks between eukaryotic phytoplankton and heterotrophic bacteria in the surface ocean. *Nature Microbiology* accepted.

Landa M, Burns AS, **Durham BP**, Esson K, Nowinski BC, Sharma S, Vorobev A, Nielsen T, Kiene RP, Moran MA. 2019. Sulfur metabolites that facilitate oceanic phytoplankton-bacteria carbon flux. *ISME Journal* epub ahead of print.  
[doi: 10.1038/s41396-019-0455-3]  
link to pdf: https://rdcu.be/bHQBR

Becker KW, Collins JR, **Durham BP**, Groussman RD, White AE, Fredricks HF, Ossolinski JE, Repeta DJ, Carini P, Armbrust EV, Van Mooy BAS. 2018. Daily changes in phytoplankton lipidomes reveal mechanisms of energy storage in the open ocean. *Nature Communications* 9: 5179.  
[doi: 10.1038/s41467-018-07346-z]

**Durham BP**, Dearth SP, Sharma S, Amin SA, Smith CB, Campagna SR, Armbrust EV, Moran MA. 2017. Recognition cascade and metabolite transfer in a marine bacteria-phytoplankton model system. *Environmental Microbiology* 19:3500-3513.  
[doi: 10.1111/1462-2920.13834]

Amin SA, Hmelo LR, van Tol HM, **Durham BP**, Carlson LT, Heal KR, Morales RL, Berthiaume CT, Parker MS, Djunaedi B, Ingalls AE, Parsek MR, Moran MA, Armburst EV. 2015. Interaction and signalling between a cosmopolitan phytoplankton and associated bacteria. *Nature* 522:98-101.  
[doi: 10.1038/nature14488]

Varaljay VA, Robidart J, Preston CM, Gifford SM, **Durham BP**, Burns AS, Ryan JP, Marin R III, Kiene RP, Zehr JP, Scholin CA, Moran MA. 2015. Single-taxon field measurements of bacterial gene regulation controlling DMSP fate. *ISME Journal* 9:1677-1686.  
[doi: 10.1038/ismej.2015.23]

**Durham BP**, Sharma S, Luo H, Smith CB, Amin SA, Bender SJ, Dearth SP, Van Mooy BAS, Campagna SR, Kujawinski EB, Armbrust EV, Moran MA. 2015. Cryptic carbon and sulfur cycling between surface ocean plankton. *Proc Natl Acad Sci USA* 112:453-457.  
[doi: 10.1073/pnas.1413137112]

**Durham BP**, Grote J, Whittaker KA, Bender SJ, Luo H, Grim SL, Brown JM, Casey JR, Dron A, Florez-Leiva L, Krupke A, Luria CM, Mine AH, Pather S, Talarmin A, Wear EK, Weber TS, Wilson JM, Church MJ, DeLong EF, Karl DM, Steward GF, Eppley JM, Kyrpides NC, Schuster S, Rapp√© MS. 2014. Draft genome sequence of marine alphaproteobacterial strain HIMB11, the first cultivated representative of a unique lineage within the Roseobacter clade possessing a remarkably small genome. *Standards in Genomic Sciences* 9:632-645.  
[doi: 10.4056/sigs.4998989]

Moran MA, Satinsky B, Gifford SM, Luo H, Rivers A, Chan LK, Meng J, **Durham BP**, Shen C, Varaljay VA, Smith CB, Yager PL, Hopkinson BM. 2013. Sizing up metatranscriptomics. *ISME Journal* 7:237-243.  
[doi: 10.1038/ismej.2012.94]
